imap_processing/swe/l2/swe_l2.py

```diff
@@ -0,0 +1,31 @@
+"""SWE L2 processing module."""
+
+import numpy.typing as npt
+
+from imap_processing.swe.l1b.swe_l1b_science import read_lookup_table
+
+
+def calculate_particle_energy(esa_table_num: int) -> npt.NDArray:
+    """
+    Calculate particle energy.
+
+    To convert Volts to Energy, multiply ESA voltage in Volts by 4.75 to get
+    electron energy in eV.
+
+    Parameters
+    ----------
+    esa_table_num : int
+        ESA table number.
+
+    Returns
+    -------
+    energy : np.ndarray
+        720 step energy values.
+    """
+    # The lookup table gives voltage applied to analyzers.
+    esa_table = read_lookup_table(table_index_value=esa_table_num)
+
+    # Convert voltage to electron energy in eV by apply conversion factor.
+    conversion_factor = 4.75
+    energy = esa_table["esa_v"] * conversion_factor
```

Comment:
It looks like this is applying a constant conversion to the esa table values. If that's the case, these values could be added as a new column to the table and pulled directly during processing instead of multiplying this constant conversion factor every time. Although I guess the problem with changing the file is needing to document that somewhere to do it again if the tables change some day.

Reply:
Good point. Unfortunately, they will likely add or change or update lookup tables and `esa_v` after launch. I was told that `esa_v` could be ordered differently and so on. If that, then keeping this here makes it easier at the moment.